src/transformers/modeling_utils.py

```diff
@@ -1968,6 +1977,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         load_in_8bit = kwargs.pop("load_in_8bit", False)
         load_in_8bit_threshold = kwargs.pop("load_in_8bit_threshold", 6.0)
         load_in_8bit_skip_modules = kwargs.pop("load_in_8bit_skip_modules", None)
+        keep_in_fp32_modules = kwargs.pop("keep_in_fp32_modules", None)
```

Comment:
This should be a class attribute on the model class, not yet another kwarg in `from_pretrained`.

Reply:
Agreed and fixed in 2bd00b3 - but I think we still need a kwarg here, imagine if a user does not want to upcast these modules?

Reply:
Or we can just enable it if `torch_dtype != torch.float32`

Reply:
They can just adapt the class variable in that case, but they shouldn't not do this, that's the whole point.

Reply:
I see, I am convinced now!